api/handlers.go

```diff
@@ -662,6 +655,13 @@ func (s *Server) ObjectLifeline(ctx echo.Context, objectReference server.ObjectR
 	})
 }
 
+func (s *Server) findEdgePNInJetDrops(jetDrops []models.JetDrop, sortByAsc bool) (int64, int64) {
+	if sortByAsc {
+		return jetDrops[0].PulseNumber, jetDrops[len(jetDrops)-1].PulseNumber
```

Comment:
this function is not testable, I understand why you are so sure that jetDrops is not empty, but the function should be idempotent. as I said, the function is not idempotent.

Reply:
but are you sure about "idempotency" term?
I'll add a check, but I think it's not needed here, early exit is upper